src/components/CartPage/CartPage.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import cx from 'classnames';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import CartItem from './CartItem';
+import config from '../../../config';
+
+import './CartPage.scss';
+
+const CN = 'cart-page';
+
+const CartPage = (props) => {
+  const {
+    className, incrementQuantity, decrementQuantity, removeItemFromCart, userCart
+  } = props;
+  const { cartTableHeadings } = config;
+  const products = [
+    {
+      id: 10,
+      title: 'beautiful swing coat',
+      image: 'https://static.massimodutti.net/3/photos/2019/I/0/1/p/6406/512/800/6406512800_2_2_16.jpg?t=1571072186173&impolicy=massimodutti-itxmediumhigh&imwidth=900',
+      color: 'blue',
+      size: 12,
+      price: 40,
+      inventory: 10
+    },
+    {
+      id: 12,
+      title: 'nice swingy jacket',
+      image: 'https://static.massimodutti.net/3/photos/2019/I/0/1/p/6747/710/808/6747710808_1_1_16.jpg?t=1571170096577&impolicy=massimodutti-itxmediumhigh&imwidth=600',
+      color: 'green',
+      size: 12,
+      price: 60,
+      inventory: 7
+    }
+  ];
+
+  // these products are hard-coded here, as soon as wishlist is merged, products will come from the store. //
+
+  const deliveryCost = 35;
+  const voucher = 5;
+
+  const checkIfExists = (item) => userCart.find((cartItem) => item.id === cartItem.id);
```

Comment:
can be replaced with:
[Array.prototype.includes](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
**checkIfExists** not very descriptive name, exist what exactly?
**isItemInCart** is better naming for example, if you convert it to bool also

Reply:
it is impossible in this case^ 
Since i've got not only ids in my userCart

Reply:
I've got you